<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shangyedaikuan</class>
 <widget class="QWidget" name="shangyedaikuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>根据面积、单价计算</string>
  </property>
  <widget class="QPushButton" name="pushButton_kaishiJisuan">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>元/平米</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>平方米</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>按揭年数：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_anjieNianshu">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>还款方式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_huankuanFangshi">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>等额本息  </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等额本金  </string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_daikuanLilv">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款利率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>按揭成数：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_anjieChengshu">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fangwuMianji">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>房屋面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>房屋单价：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fangwuDanjia">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>温馨提示：请您输入有意义的数字！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_fangwuDanjia</sender>
   <signal>textChanged(QString)</signal>
   <receiver>shangyedaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_fangwuMianji</sender>
   <signal>textChanged(QString)</signal>
   <receiver>shangyedaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_daikuanLilv</sender>
   <signal>textChanged(QString)</signal>
   <receiver>shangyedaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_text_changed()</slot>
 </slots>
</ui>
